nerdlandbot/commands/dadJoke.py

```diff
@@ -0,0 +1,19 @@
+import discord
+import requests
+
+from discord.ext import commands
+
+
+class DadJoke(commands.Cog, name="dadJoke"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="dadJoke", help="dadjoke_help")
+    async def cmd_dadJoke(self, ctx: commands.Context): 
+        r = requests.get('https://icanhazdadjoke.com',headers = { "Accept": "text/plain" })
```

Comment:
Unfortunately we cannot use the requests module, due to the asynchronous environment we're in.
Rather you should use the aoihttp module, example can be found [here](https://docs.aiohttp.org/en/stable/client_quickstart.html#make-a-request)

Reply:
Priobably a typo, but it's AIOHTTP (Asynchronous Input Output HTTP)